hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/HandleWorkflowTest.java

```diff
@@ -180,7 +186,8 @@ void setup(@Mock InstantSource instantSource) {
                 checker,
                 serviceLookup,
                 configProvider,
-                instantSource);
+                instantSource,
+                recordCache);
     }
 
     @SuppressWarnings("ConstantConditions")
```

Comment:
Consider this alternative - better or worse (or no different)?

```
    @Test
    void testContructorFailsWithAnyNullArgument() {
        final var softly = new SoftAssertions();

        final var instantSource = InstantSource.system();
        final var args = List.of(
                networkInfo,
                preHandleWorkflow,
                dispatcher,
                blockRecordManager,
                signatureExpander,
                signatureVerifier,
                checker,
                serviceLookup,
                configProvider,
                instantSource,
                recordCache);
        for (int i = 0; i < args.size(); i++) {
            final var thisTestArgs = new ArrayList<>(args);
            thisTestArgs.set(i, null);
            softly.assertThatNullPointerException().as("throws if argument %d is null".formatted(i))
                    .isThrownBy(() -> new HandleWorkflow(
                            (NetworkInfo) thisTestArgs.get(0),
                            (PreHandleWorkflow) thisTestArgs.get(1),
                            (TransactionDispatcher) thisTestArgs.get(2),
                            (BlockRecordManager) thisTestArgs.get(3),
                            (SignatureExpander) thisTestArgs.get(4),
                            (SignatureVerifier) thisTestArgs.get(5),
                            (TransactionChecker) thisTestArgs.get(6),
                            (ServiceScopeLookup) thisTestArgs.get(7),
                            (ConfigProvider) thisTestArgs.get(8),
                            (InstantSource) thisTestArgs.get(9),
                            (HederaRecordCache) thisTestArgs.get(10)
                    ));
        }
        softly.assertAll();
    }
```

Reply:
Not sure. Maybe we should have a generic utility method that uses reflection to invoke the constructor with null in all the places where null is not allowed, and reuse it everywhere. Then it would be a single line of code from each test that needs it.

Reply:
I tried reflection but felt it was too obscure/hard to read when it all got put together.  But if this kind of thing happened in more than one place and the procedure was stuck in a utility method for all, that'd be different.